scipy/sparse/linalg/_eigen/_svds.py

```diff
@@ -287,9 +288,9 @@ def matmat_XH_X(x):
             X = np.reshape(v0, (-1, 1))
         else:
             if rs_was_None:
-                X = np.random.RandomState(52).randn(min(A.shape), k)
+                X = np.random.RandomState(lrs).randn(min(A.shape), k)
```

Comment:
`standart_normal` should be used as `randn` is not advised in new code.

Also since this would break backward compatibility, we should use our RNG helper function from `_utils`. Fixing the seed to an arbitrary number like this is an abomination we should never have had 😅